.github/workflows/criteo-cookbooks-ci.yml

```diff
@@ -0,0 +1,45 @@
+# This is a basic workflow to help you get started with Actions
+name: Criteo Cookbooks CI
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
```

Comment:
Please add a `supermarket` job (See what we have done on other criteo-cookbooks like [choregraphie](https://github.com/criteo-cookbooks/choregraphie), [gobgp](https://github.com/criteo-cookbooks/gobgp) or [nexus3](https://github.com/criteo-cookbooks/nexus3)